packages/fungible-token/bridge-fungible-token/src/main.sw

```diff
@@ -97,7 +98,7 @@ impl MessageReceiver for Contract {
             return;
         };
 
-        let res_amount = adjust_deposit_decimals(message_data.amount, DECIMALS, BRIDGED_TOKEN_DECIMALS);
+        let res_amount = adjust_deposit_decimals(message_data.amount, DECIMALS.try_as_u8().unwrap(), BRIDGED_TOKEN_DECIMALS.try_as_u8().unwrap());
```

Comment:
`DECIMALS` and `BRIDGED_TOKEN_DECIMALS` are constants so this SHOULD never revert but `unwrap_or()` would enforce this

Reply:
Committed suggestion to use a default value and applied `forc fmt`